.travis.yml

```diff
@@ -4,7 +4,7 @@ env:
   - MOZ_HEADLESS=1
 
 addons:
-  firefox: "55.0"
+  firefox: latest
```

Comment:
is this intentional? if the newer versions of firefox introduce some incompatible changes, builds will start failing.

Reply:
Umm... actually 55 had bugs related to selenium fixed in 57 but even after bumping the version the build is failing for resume upload part.

Reply:
I would prefer that we get to know ASAP when something breaks and fix it and stay up to date